tests/phpunit/src/ApplicationTestBase.php

```diff
@@ -25,9 +25,6 @@ class ApplicationTestBase extends TestBase {
 
   public function setUp($output = NULL):void {
     parent::setUp($output);
-    // If kernel is cached from a previous run, it doesn't get detected in code
```

Comment:
Not sure why but this seems no longer relevant. You can see our coverage stays the same in this PR.